### PR TITLE
refactoring: delete date service logic, merged function of service in…

### PR DESCRIPTION
**Pull request by leleOFA**
… date util file and substitute import and usage of utils date function. Also change logic in formatDateTime with object, changed with return of string

## Short description
i refactor the all date logic in service and utility  

## List of changes proposed in this pull request
- rename function with get in the name and substitute with format
- change return of formatDateTime from object to string contain this date format DD/MM/YY, ore ss/mm/hh

### CODE REVIEW

**Review by AndreaCimini90 — DISMISSED** [review]

File: packages/pn-commons/src/utils/__test__/date.utility.test.ts [hunk 1]
```diff
@@ -1,11 +1,17 @@
-import { getMonthString, getDay, getTime, getNextDay, formatToTimezoneString } from '../date.utility';
+import {
```
[the remark this answers is not in the document]
Comment (leleOFA) [reply packages/pn-commons/src/utils/__test__/date.utility.test.ts:1]: do it

File: packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx [hunk 2]
```diff
@@ -4,36 +4,38 @@ import DownloadIcon from '@mui/icons-material/Download';
...
   if (date) {
     const dateAndTime = formatDateTime(date);
-    return inTwoLines
-      ? <Stack direction="column">
-        <Typography variant="body2">{dateAndTime.date},</Typography>
-        <Typography variant="body2">{dateAndTime.time}</Typography>
+    return inTwoLines ? (
+      <Stack direction="column">
+        <Typography variant="body2">{formatDate(date)},</Typography>
+        <Typography variant="body2">{formatTime(date)}</Typography>
```
Comment (clombardi) [inline packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx:19]: A bug slipped here: the second line lacks the indication 'ore'
image
which is present (correctly) in the mobile version
image
I suggest adding a function `formatTimeWithLegend` which prepends the `date-time.hour-of-day` text to the time in HH:MM format.

File: packages/pn-commons/src/utils/__test__/date.utility.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (clombardi) [reply packages/pn-commons/src/utils/__test__/date.utility.test.ts:1]: If you decide to create the function `formatTimeWithLegend` as suggested in another comment, I would ask you to add a test for such function as well.

File: packages/pn-personafisica-webapp/src/pages/__test__/AppStatus.page.test.tsx [hunk 3]
```diff
@@ -4,14 +4,13 @@ import {
...
   formatDate,
-  formatTimeHHMM,
   KnownFunctionality,
   ResponseEventDispatcher,
 } from '@pagopa-pn/pn-commons';
 import { render } from '../../__test__/test-utils';
 import AppStatus from '../AppStatus.page';
 import { APP_STATUS_ACTIONS } from '../../redux/appStatus/actions';
-
+import { formatTime } from '../../../../pn-commons/src/utils';
```
Comment (clombardi) [inline packages/pn-personafisica-webapp/src/pages/__test__/AppStatus.page.test.tsx:13]: **beware** this import should be from `@pagopa-pn/pn-commons`!!

**Review by clombardi — CHANGES_REQUESTED** [review]
Please resolve the comments included and re-request review, the modifications inserted a little bug, and there are some technical corrections to be done as well.

File: packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx [hunk 4]
```diff
@@ -4,36 +4,38 @@ import DownloadIcon from '@mui/icons-material/Download';
...
       </Stack>
-      : <Typography variant="body2">{dateAndTime.date}, {dateAndTime.time}</Typography>;
+    ) : (
+      <Typography variant="body2">{dateAndTime}</Typography>
```
[the remark this answers is not in the document]
Comment (leleOFA) [reply packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx:22]: these two lines have already been modified, I deleted the variable in the else and used the dateTime function

File: packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx:19]: create function suggested

File: packages/pn-personafisica-webapp/src/pages/__test__/AppStatus.page.test.tsx (hunk 3 above)
Comment (leleOFA) [reply packages/pn-personafisica-webapp/src/pages/__test__/AppStatus.page.test.tsx]: do it

**Review by clombardi — APPROVED** [review]
In my opinion, this is ready to merge.

**Pull request merged**